### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by Zuzakiin** [anapnoe/stable-diffusion-webui-ux#30]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What happened?

Changes to Hidden Tabs and Header Tabs has immediate effect, but does not propagate across reloads even when the Apply Settings is used and ' settings changed: ui_header_tabs ' is displayed as confirmation

In other words, if http://127.0.0.1:7860 is reloaded, the default of all the possible tabs shows up.
the check boxes are ticked properly when going back to that part of the menu, but the settings are ignored until (any hidden or header) check box is modified again

### Steps to reproduce the problem

click settings
click User interface
go to Header Tabs check boxes and tick some as desired
immediate effect. 
click on Apply Settings,
displayed below apply settings: settings changed: ui_header_tabs.

 reload UI
when reloaded, too many tabs, ticks ignored.

### What should have happened?

hidden tab settings should take effect on page / UI load without going into settings>User interface and tick/untick something for each and every instance  of the UI to make the tabs stop overlapping.

### Commit where the problem happens

608c684

### What platforms do you use to access the UI ?

Windows

### What browsers do you use to access the UI ?

Mozilla Firefox, Google Chrome

### Command Line Arguments

```Shell
set COMMANDLINE_ARGS=  --xformers --opt-channelslast
```

### List of extensions

ABG_extension	https://github.com/KutsuyaYuki/ABG_extension.git	3e60da27 (Wed Mar 29 08:27:53 2023)	unknown
Config-Presets	https://github.com/Zyin055/Config-Presets.git	1b952e26 (Tue Mar 7 03:53:07 2023)	unknown
SD-latent-mirroring	https://github.com/dfaker/SD-latent-mirroring	05df9e83 (Sat Mar 25 14:37:33 2023)	unknown
a1111-sd-webui-haku-img	https://github.com/KohakuBlueleaf/a1111-sd-webui-haku-img.git	99344e7f (Fri Feb 10 14:33:00 2023)	unknown
a1111-sd-webui-locon	https://github.com/KohakuBlueleaf/a1111-sd-webui-locon.git	8e0ebd76 (Thu Mar 30 15:20:06 2023)	unknown
a1111-sd-webui-tagcomplete	https://github.com/DominikDoom/a1111-sd-webui-tagcomplete	811d4622 (Tue Mar 28 18:49:09 2023)	unknown
a1111-stable-diffusion-webui-vram-estimator	https://github.com/space-nuko/a1111-stable-diffusion-webui-vram-estimator.git	bf8797a3 (Sun Mar 5 03:42:33 2023)	unknown
animator_extension	https://github.com/Animator-Anon/animator_extension	88a15bd2 (Sat Mar 4 09:16:05 2023)	unknown
asymmetric-tiling-sd-webui	https://github.com/tjm35/asymmetric-tiling-sd-webui.git	b4470ddc (Fri Nov 18 17:42:54 2022)	unknown
auto-sd-paint-ext	https://github.com/Interpause/auto-sd-paint-ext	272e1847 (Sat Mar 25 07:58:56 2023)	unknown
batch-face-swap	https://github.com/kex0/batch-face-swap.git	60e259d8 (Thu Mar 16 02:02:46 2023)	unknown
canvas-zoom	https://github.com/richrobber2/canvas-zoom.git	fd5d8a13 (Thu Mar 30 20:51:00 2023)	unknown
custom-diffusion-webui	https://github.com/guaneec/custom-diffusion-webui.git	017757d2 (Sun Feb 5 07:08:07 2023)	unknown
ddetailer	https://github.com/dustysys/ddetailer.git	22a17414 (Mon Nov 28 09:33:31 2022)	unknown
depth-image-io-for-SDWebui	https://github.com/AnonymousCervine/depth-image-io-for-SDWebui.git	9bfe4a0b (Thu Jan 12 18:59:31 2023)	unknown
depthmap2mask	https://github.com/Extraltodeus/depthmap2mask.git	7eedf6af (Mon Mar 20 10:16:05 2023)	unknown
ebsynth_utility	https://github.com/s9roll7/ebsynth_utility.git	1e91627c (Sun Mar 19 05:20:43 2023)	unknown
gif2gif	https://github.com/LonicaMewinsky/gif2gif.git	230f6737 (Wed Mar 15 21:51:11 2023)	unknown
multi-subject-render	https://github.com/Extraltodeus/multi-subject-render.git	03427e26 (Mon Mar 6 14:11:30 2023)	unknown
novelai-2-local-prompt	https://github.com/animerl/novelai-2-local-prompt	0e03ca1a (Sat Jan 28 06:33:03 2023)	unknown
openOutpaint-webUI-extension	https://github.com/zero01101/openOutpaint-webUI-extension	566a8a53 (Thu Mar 30 22:50:34 2023)	unknown
openpose-editor	https://github.com/fkunn1326/openpose-editor.git	a63fefc3 (Thu Mar 30 08:11:41 2023)	unknown
posex	https://github.com/hnmr293/posex.git	233daf8e (Tue Mar 7 12:21:57 2023)	unknown
sd-3dmodel-loader	https://github.com/jtydhr88/sd-3dmodel-loader.git	8a1ec0ad (Thu Mar 30 01:34:51 2023)	unknown
sd-dynamic-prompts	https://github.com/adieyal/sd-dynamic-prompts	b020cb48 (Thu Mar 30 16:54:54 2023)	unknown
sd-dynamic-thresholding	https://github.com/mcmonkeyprojects/sd-dynamic-thresholding.git	8f18058b (Mon Mar 27 19:50:18 2023)	unknown
sd-extension-aesthetic-scorer	https://github.com/vladmandic/sd-extension-aesthetic-scorer	250871ed (Tue Mar 28 13:20:09 2023)	unknown
sd-extension-system-info	https://github.com/vladmandic/sd-extension-system-info	98051500 (Fri Mar 31 06:18:29 2023)	unknown
sd-webui-3d-open-pose-editor	https://github.com/nonnonstop/sd-webui-3d-open-pose-editor.git	99f3e80d (Tue Mar 28 11:58:16 2023)	unknown
sd-webui-ar	https://github.com/alemelis/sd-webui-ar.git	4ac84484 (Thu Mar 30 19:42:57 2023)	unknown
sd-webui-aspect-ratio-helper	https://github.com/thomasasfk/sd-webui-aspect-ratio-helper.git	36c45011 (Sun Mar 26 23:06:03 2023)	unknown
sd-webui-controlnet	https://github.com/Mikubill/sd-webui-controlnet	241c05f8 (Thu Mar 23 15:18:35 2023)	unknown
sd-webui-enable-checker	https://github.com/shirayu/sd-webui-enable-checker.git	3080c440 (Tue Mar 28 01:08:08 2023)	unknown
sd-webui-gelbooru-prompt	https://github.com/antis0007/sd-webui-gelbooru-prompt.git	5a5ff86c (Tue Mar 7 18:11:17 2023)	unknown
sd-webui-latent-couple	https://github.com/miZyind/sd-webui-latent-couple	aa7cc980 (Sat Mar 4 16:04:02 2023)	unknown
sd-webui-llul	https://github.com/hnmr293/sd-webui-llul.git	b20337ae (Wed Mar 22 15:56:08 2023)	unknown
sd-webui-lora-block-weight	https://github.com/hako-mikan/sd-webui-lora-block-weight.git	a6eaf753 (Fri Mar 31 03:31:40 2023)	unknown
sd-webui-model-converter	https://github.com/Akegarasu/sd-webui-model-converter.git	d19e2816 (Sun Mar 26 06:36:49 2023)	unknown
sd-webui-modelscope-text2video	https://github.com/deforum-art/sd-webui-modelscope-text2video.git	d16d9d47 (Thu Mar 30 09:18:56 2023)	unknown
sd_grid_add_image_number	https://github.com/AlUlkesh/sd_grid_add_image_number.git	453c305d (Thu Feb 23 01:08:51 2023)	unknown
sd_smartprocess	https://github.com/d8ahazard/sd_smartprocess	41fb35ef (Sun Jan 22 17:30:02 2023)	unknown
sd_web_ui_preset_utils	https://github.com/Gerschel/sd_web_ui_preset_utils.git	175b6f7d (Mon Jan 30 15:07:50 2023)	unknown
sdweb-merge-board	https://github.com/bbc-mc/sdweb-merge-board.git	f59c60fd (Wed Jan 25 15:00:09 2023)	unknown
seed_travel	https://github.com/yownas/seed_travel.git	59973b8d (Thu Mar 30 20:31:45 2023)	unknown
shift-attention	https://github.com/yownas/shift-attention.git	6dc140f5 (Sat Mar 18 20:03:29 2023)	unknown
stable-diffusion-webui-anti-burn	https://github.com/klimaleksus/stable-diffusion-webui-anti-burn.git	4d678f1f (Sun Feb 19 19:22:44 2023)	unknown
stable-diffusion-webui-cafe-aesthetic	https://github.com/p1atdev/stable-diffusion-webui-cafe-aesthetic.git	d1c7ee84 (Sun Mar 26 13:36:41 2023)	unknown
stable-diffusion-webui-composable-lora	https://github.com/opparco/stable-diffusion-webui-composable-lora.git	d4963e48 (Mon Feb 27 17:40:08 2023)	unknown
stable-diffusion-webui-daam	https://github.com/kousw/stable-diffusion-webui-daam.git	796acea1 (Sun Dec 18 13:36:12 2022)	unknown
stable-diffusion-webui-dumpunet	https://github.com/hnmr293/stable-diffusion-webui-dumpunet.git	c6d22b9d (Fri Mar 24 04:12:32 2023)	unknown
stable-diffusion-webui-images-browser	https://github.com/yfszzx/stable-diffusion-webui-images-browser	a42c7a30 (Sat Dec 10 11:01:30 2022)	unknown
stable-diffusion-webui-model-toolkit	https://github.com/arenatemp/stable-diffusion-webui-model-toolkit	8d8f073f (Fri Feb 17 20:39:41 2023)	unknown
stable-diffusion-webui-prompt-travel	https://github.com/Kahsolt/stable-diffusion-webui-prompt-travel.git	771cb5c0 (Thu Mar 30 20:02:00 2023)	unknown
stable-diffusion-webui-promptgen	https://github.com/AUTOMATIC1111/stable-diffusion-webui-promptgen.git	84e58b5d (Fri Jan 20 11:15:12 2023)	unknown
stable-diffusion-webui-rembg	https://github.com/AUTOMATIC1111/stable-diffusion-webui-rembg.git	64821f04 (Sun Mar 12 15:32:00 2023)	unknown
stable-diffusion-webui-sonar	https://github.com/Kahsolt/stable-diffusion-webui-sonar.git	6cf2cf3f (Thu Mar 9 11:50:57 2023)	unknown
stable-diffusion-webui-state	https://github.com/ilian6806/stable-diffusion-webui-state.git	50ff7bb3 (Thu Mar 30 13:11:50 2023)	unknown
stable-diffusion-webui-tokenizer	https://github.com/AUTOMATIC1111/stable-diffusion-webui-tokenizer.git	ac6d541c (Sat Dec 10 12:58:31 2022)	unknown
video_loopback_for_webui	https://github.com/fishslot/video_loopback_for_webui	11bd5106 (Tue Mar 14 13:24:12 2023)	unknown
LDSR	built-in		
Lora	built-in		
ScuNET	built-in		
SwinIR	built-in		
prompt-bracket-checker	built-in		
sd_theme_editor	built-in

### Console logs

```Shell
To create a public link, set `share=True` in `launch()`.
auto-sd-paint-ext:WARNING: COMMANDLINE_ARGS does not contain --api, API won't be mounted.
Startup time: 75.8s (import gradio: 2.4s, import ldm: 3.6s, other imports: 2.2s, list extensions: 13.0s, setup codeformer: 0.2s, load scripts: 9.5s, load SD checkpoint: 33.3s, create ui: 11.5s, gradio launch: 0.2s).

only error, pops when testing without --api
```

### Additional information

Tested, same results with chrome, firefox, and opera.
I use multiple browser tabs and update extensions every day. It gets tedious.
I really like this interface. This is my only real gripe abut it.

**Comment by anapnoe** [comment]
I tested and can't reproduce maybe you can disable all extensions updated after 25th of March and test it again it seems that you have updated to incompatible extensions, there is no versioning system for extensions and newer extensions  may not work with this stable release, commits of extensions should be before 25 Mar 2023 if this is not the issue please report back so I have a deeper look into it

**Comment by Zuzakiin** [comment]
I really like the click menu for all the tabs.

March 25?
The issue wa a problem since I discovered this gui over ten days ago.
March... 19 ish i think.


Specifics of extensions is irrelevent besides total numbers exceeding what
can fit in webui without overapping picture or interface elements.
Hence needing to use the menu to hide many of them to get it to a row or
two.

I spent a bunch of time trying to understand styles.css and other files
that should control placements with no luck. Goal was to make picture /
interface area lower to prevent overlap. Insufficient brainz.

The obvious simple solution for overlap that does work of fewer extenstions
ya git! Defeats the desires of flexibility and minimization of
micromanagement and planning.

....

One possibilty that I havent tested is having this gui be in its own folder
instead of force overwriting automatic1111 folder. Everything else works
though.
Maybe tommorrow will get time.

**Comment by anapnoe** [comment]
Yes you should install it the other way around UI-UX as default, then copy-move your models and your extensions to the appropriate folders this is a fork and many files of A1111 are modified it uses different implementation for hidden tabs that work in parallel with header tabs and does not require restart. Since last update March 25th the original A1111 project become unstable this project will not update until things sort out many extensions have issues with the new version of A1111 and their authors try to fixed them the fixes may are not compatible to previous versions  so before you apply any update to any extension make a backup as there is not versioning for extensions there is no way to detect compatibility issues

**Comment by Zuzakiin** [comment]
After much futzing with making new installations, testing, and trying copying over old folders, I have narrowed down the root problem to, as you said, the extensions updated on March 25 and newer.
Odd because I thought the problem started before then. Shrug.

Ok, so I was mistaken and the problem is not with this excellent gui.

At least I learned a few things about symlinks and stuff in the process to minimize redundant duplication of giant venv and models folders. Like sd is ok with normal symlink for venv, but will replace models with normal folder and attempt to download 1.5 junction seems fine.
https://github.com/amd989/Symlinker

I will hold off on updating existing extensions or the now separate install of 1111 until a method is found to avoid whatever is causing the derp.
Sorry to be a bother and a fool on this foolish day.
